File: app/src/main/java/com/example/stud_informer_irkpo/network/PerformanceApi.java
package com.example.stud_informer_irkpo.network;

import com.example.stud_informer_irkpo.models.PerformanceResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface PerformanceApi {
    @GET("student/{phoneNumber}")
    Call<PerformanceResponse> getPerformance(@Path("phoneNumber") String phoneNumber);
}

File: app/src/main/java/com/example/irkpo_management/network/PerformanceApi.kt
package com.example.irkpo_management.network

import com.example.irkpo_management.models.PerformanceResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface PerformanceApi {
    @GET("student/{phoneNumber}")
    fun getPerformance(@Path("phoneNumber") phoneNumber: String?): Call<PerformanceResponse?>?
}
